rtengine/procparams.cc

```diff
@@ -129,22 +131,22 @@ RetinexParams::RetinexParams ()
     setDefaults ();
 }
 
-void RetinexParams::getDefaulttransmissionCurve(std::vector<double> &curve)
+void RetinexParams::getDefaulttransmissionCurve (std::vector<double> &curve)
```

Comment:
:man_facepalming: I blame confusing the definition for a call on my illness :stuck_out_tongue: 

I astyled those 11 files in `dev` and created branch `astyle-for-ciecamout` from that, then I astyled the same 11 files in `ciecamout` and created the branch `astyled-ciecamout` from that (sorry for the sloppy astyle-branch names). Now if you compare `git diff dev..ciecamout` to `git diff astyle-for-ciecamout..astyled-ciecamout`, the latter is cleaner.

Here is the output of `git diff astyle-for-ciecamout..astyled-ciecamout` https://paste.pound-python.org/show/FskNAAxenlGyE0kY8M8S/

Note: don't use GitHub's compare feature for that, because GitHub forces the use of `...` (shows changes on the compared branch since it was started from the reference branch) while you need to use `..` (changes between the tips of both branches) to see the clean diff.

As for the style format, I have no say in that.